src/main/java/seedu/connectus/model/tag/Tag.java

```diff
@@ -10,7 +10,8 @@
 public class Tag implements Comparable<Tag> {
 
     public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
-    public static final String VALIDATION_REGEX = "[\\p{Alnum} ]+";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum} ]+#?[\\p{Alnum}\\s]+$";
```

Comment:
I'm not sure if this is a good idea, personally I feel like it should be better to keep tags as this regex and have a separate one for CCA. I'm trying to find a workaround on it, does anyone got suggestions/opinions on this?